ext/civi_mail/ang/afsearchEmailBounceHistory.aff.html

```diff
@@ -0,0 +1,4 @@
+<div af-fieldset="">
+  <crm-search-display-table search-name="Email_Bounce_History" display-name="Email_Bounce_History_Table" filters="{'MailingEventBounce_MailingEventQueue_event_queue_id_01.id': routeParams['MailingEventBounce_MailingEventQueue_event_queue_id_01.email_id']}">
```

Comment:
It looks like this routeParam is just named `email_id` (based on the link you constructed in the tpl).
```suggestion
  <crm-search-display-table search-name="Email_Bounce_History" display-name="Email_Bounce_History_Table" filters="{'MailingEventBounce_MailingEventQueue_event_queue_id_01.id': routeParams.email_id}">
```